geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa

```diff
@@ -109,9 +109,7 @@ if  [[ $1 = configure ]]; then
   geomesaConfigure
 elif [[ $1 = classpath ]]; then
   CLASSPATH="$GEOMESA_CP:$ACCUMULO_CP:$HADOOP_CP:${ZOOKEEPER_JAR}"
-  for element in ${CLASSPATH//:/ } ; do
-    echo ${element}
-  done
+  echo $CLASSPATH | tr ':' '\n' | xargs ls -di | sort -u -k1,1 | awk '{print $2}' | sort
```

Comment:
Ah, no.  Actually, my fix is listing the actual files (which is different maybe...)

Eh, I think I'll close this PR.  The ticket might rot for a bit (unless someone else posts on it).